### PR TITLE
Build and test on Windows CI using AppVeyor

### PR DESCRIPTION
**Pull request by ellenmobify**
A first pass at using [AppVeyor](https://ci.appveyor.com/projects) to install, build, and test on a Windows environment.

## Changes
- Add `appveyor.yml` config

## How to test-drive this PR
- Examine AppVeyor build output and ensure that there are no errors, eg https://ci.appveyor.com/project/ellenmobify/platform-scaffold/build/1.0.63
- Verify CircleCI Builds are not affected

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+cache:
+  - web/node_modules
+  - native/node_modules
+
+# Install scripts. (runs after repo cloning)
+install:
+  # Get the latest stable version of Node.js or io.js
+  - ps: Install-Product node $env:nodejs_version
+  # install modules
+  - pushd web && npm install && popd web
```
Comment (jeremywiebe) [inline appveyor.yml]: I don't think `popd` requires any parameter. It's just popping off the `pushd` stack.

**Comment by jeremywiebe** [comment]
Code looks fine, but the builds on AppVeyor look to be failing.  Is it possible to fix them before this PR is done?

**Comment by ellenmobify** [comment]
Thanks for reviewing @jeremywiebe! Builds are green now.

**Comment by ellenmobify** [comment]
🤔 Will need to look into how AppVeyor integrates with GitHub. It seems to switch back and forth between the build versions, showing failure on an old build (https://ci.appveyor.com/project/johnboxall/platform-scaffold/build/1.0.17) and success on latest (https://ci.appveyor.com/project/ellenmobify/platform-scaffold). Maybe we need the `mobify` user to add the project?

**Pull request merged**